redaxo/src/core/lib/response.php

```diff
@@ -483,4 +483,10 @@ private static function md5($content)
     {
         return md5(preg_replace('@<!--DYN-->.*<!--/DYN-->@U', '', $content));
     }
+
+    public static function enforceHttps()
+    {
+        self::setStatus(self::HTTP_MOVED_PERMANENTLY);
+        self::sendRedirect('https://'.$_SERVER['SERVER_NAME'].$_SERVER['REQUEST_URI']);
```

Comment:
Ich glaube, den `isHttps()` check würde ich mit in diese Methode nehmen. Ich würde erwarten dass ich die Methode gefahrlos auch dann aufrufen kann, wenn https schon aktiv.